docs/config.md

```diff
@@ -134,10 +145,13 @@ Important environment variables include:
 | Environment variable | Description | Default value |
 |-|-|-|
 | `OTEL_DOTNET_AUTO_LOAD_TRACER_AT_STARTUP` | Whether the tracer is created by the automatic instrumentation library or not. Set to `false` when the application initializes the OpenTelemetry .NET SDK Tracer on its own. This configuration can be used, for example, to retrieve the bytecode instrumentations. | `true` |
+| `OTEL_DOTNET_AUTO_LOAD_METER_AT_STARTUP` | Whether the meter is created by the automatic instrumentation library or not. Set to `false` when the application initializes the OpenTelemetry .NET SDK Meter on its own. This configuration can be used, for example, to retrieve the bytecode instrumentations. | `true` |
```

Comment:
We mention `This configuration can be used, for example, to retrieve the bytecode instrumentations.` in the equivalent setting for tracing, but I'm confused by it. Maybe we should remove it?

Reply:
I agree, we should remove it. This is kept for future use.

Reply:
Fixed 9f1a571d2878ee3bc343b579ff3b343ffd30b87d